### DISCUSSION TITLE
20240201-inflab-iac/

### DISCUSSION THREAD

**Discussion by giscus** [inflearn/tech-blog-comment#48] [Comments]
# 20240201-inflab-iac/

https://tech.inflab.com/20240201-inflab-iac/

**Comment by Kwonkunkun** [comment]
와.. 잘봤습니다!

**Comment by ruseel** [comment]
염치없지만 inflab 의 명성에 기대서 널리 알려질 이글에 제가 적은 tutorial 도 광고를 해봅니다. 

   Pulumi 로 EKS 구성하기 - https://rsl.kr/posts/pulumi-eks

Pulumi 의 점유율과 평판이 무럭무럭 자라나면 좋겠습니다.

**Comment by DonggeunYu** [comment]
잘 봤습니다!

**Comment by ghkdqhrbals** [comment]
정말 잘 봤습니다!
